.golangci.yml

```diff
@@ -62,6 +62,8 @@ linters:
     - structcheck
     - ineffassign
     - misspell
+    - exportloopref
+    - unconvert
```

Comment:
I'd probably lean against including unconvert, despite my temptation to scratch the itch to fix this every time my IDE points one out - it feels like a style thing whose value would be outweighed by annoying CI failures.

Or maybe I'm in the minority in that I only rarely run lint locally rather than relying on CI.

Reply:
That's fair! Removed from this PR in 1c96f61. I really wanted the `exportloopref` linter and noticed `unconvert` in the list, which personally annoys me (due to the IDE squiggles), but yeah, it's totally a stylistic thing rather than indicative of a bug so I don't _really_ care 😛